app/views/govuk_publishing_components/components/_layout_super_navigation_header.html.erb

```diff
@@ -233,12 +184,12 @@
         hidden
         id="super-search-menu-toggle"
         type="button"
-        data-ga4-event="<%= {
+        data-ga4="<%= {
```

Comment:
Please change this from `ga4` to `ga4_event` as we have changed the attribute name

Reply:
have changed this